### ISSUE TITLE
Informing about React.lazy() 

### ISSUE THREAD

**Issue by iddan** [ctrlplusb/react-async-component#95]
This week the React team introduced a new library feature: `React.lazy()` which replicate this library functionality by using newly introduced React concept: React Suspense for promise resolution and error boundaries for error management.
react-async-component has been a wonderful solution for my projects in the last year and I'm grateful for that. Now that it's solution has been integrated into the React core I think it would be suiting to inform users that use latest React that they can use `lazy()`.

**Comment by mschipperheyn** [comment]
lazy doesn't support SSR yet

**Comment by iddan** [comment]
True, yet for non SSR use and future use when it will be supported it should be informed.
